bin/exit/main.go

```diff
@@ -63,8 +63,9 @@ func main() {
 	}
 
 	if config.Reverse {
-		for serviceName := range config.Services {
-			e := tuna.NewTunaExit(config, services, wallet)
+		for serviceName, serviceInfo := range config.Services {
+			reverseEntryToExitPrice, reverseExitToEntryPrice, err := tuna.ParsePrice(serviceInfo.Price)
```

Comment:
This is still not fixed. You should parse `ReverseMaxPrice` instead of `serviceInfo.Price`. `ReverseMaxPrice` is the highest price exit will accept, `serviceInfo.Price` is the price exit will charge when exit is service provider (not reverse mode).